server-src/src/test/java/com/itt/test/kmt/services/UserServiceTests.java

```diff
@@ -89,15 +89,15 @@ public final void getUserByEmail() {
     }
 
     @Test
-    public final void getUserById() {
+    public final void getUserByID() {
 
         // Arrange
         User user1 = testDataRepository.getUsers()
                 .get("user-1");
```

Comment:
Why are we using "user-1" and why the name of variable is "user1", we need not to use numeric in variable name here.

Reply:
Hi Shiva,
I have asked Rakshit to address your comments on variable name and keep user-1 for now.